### PR TITLE
Feature/setup curriculum

### PR DESCRIPTION
**Pull request by kamisakihideyoshi**
Just a simple collection view, with cells to display courses.
Needs to add functionality to check whether Saturday and Sunday have any course, after curriculum api updated.

### CODE REVIEW

**Comment by freddy5566** [comment]
do we need to fix up the issue that @kamisakihideyoshi mentioned above, or we open another branch to fix it?

File: TAT/Models/Course.swift [hunk 1]
```diff
@@ -40,10 +43,13 @@ class Course: NSObject, Codable {
       #endif
     }
   }
+
```
Comment (kamisakihideyoshi) [inline TAT/Models/Course.swift:46]: Should I leave a blank line above last right brace and down before everything in a class, or just personal preference, like these:
``` swift
class Foo {

  // MARK: - Bar

}
```
and 
``` swift
func foo() {
  while bar {
    // some codes here
  }

}
```

**Comment by kamisakihideyoshi** [comment]
> do we need to fix up the issue that @kamisakihideyoshi mentioned above, or we open another branch to fix it?

I'd rather choose creating new branch later on, for API format changes and new layout.
Before we come up with a custom layout or API updates, there's nothing to do here.

File: TAT/Models/Course.swift (hunk 1 above)
Comment (freddy5566) [reply TAT/Models/Course.swift:46]: yes you should do this, also leave a blank line in the vary end of class, extension, and struct.

**Review by freddy5566 — APPROVED** [review]
It seems like we are good to go, after this, I'll upgrade swift 4.2 to swift 5.
here is a repo that you can read https://github.com/twostraws/whats-new-in-swift-5-0 for a better understanding.

**Pull request merged**